LCLSGeneral/LCLSGeneral/GVLs/GeneralConstants.TcGVL

```diff
@@ -1,11 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
-  <GVL Name="GeneralConstants" Id="{57f2f1ff-0c58-4e70-a147-fb0d6c2bb252}">
+  <GVL Name="GeneralConstants" Id="{5fbde5e2-29c8-4193-ba17-69759df29f93}" ParameterList="True">
     <Declaration><![CDATA[{attribute 'qualified_only'}
 VAR_GLOBAL CONSTANT
-    // 16 including "Unknown" is the max for an EPICS MBBI
+    // 16 including "Unknown" is the max for an EPICS MBBI/MBBO
     // This is the max number of user-defined states (OUT, TARGET1, YAG...)
-    MAX_STATES: INT := 15;
+    // You can make this smaller if you want to use less memory in your program in exchange for limiting your max state count
+    // You can make this larger if you want to use states-based FBs sized beyond the EPICS enum limit
+    MAX_STATES: UINT := 15;
```

Comment:
Is there any spot in lcls-twincat-general where making an assertion that MAX_STATES is >=2, <=15 would make sense?

Reply:
I think this actually isn't included in any general lib code, but is maintained here in case non-motion libraries want to allocate state-sized arrays.

Maybe in motion lib it makes sense to do the >=2 check.
I can't think of any place where >15 causes a technical issue aside from the enum count.

Reply:
That's fair - something I hadn't considered, really. pytmc will assuredly fail on generating the record, but I suppose there's no strict requirement that states needs to be addressable by EPICS.

Reply:
Yeah, for those state movers with 20 states or w/e, we'll just use a non-MBB record.
Nothing in the motion code even cares how big the state array is- it requires that the user provide the get/set PVs themselves, for example.

Reply:
I was going to suggest patching those up in Python via https://github.com/pcdshub/pcdsdevices/blob/5c361e6c897a8f707fb91acaad790e44cbba3006/pcdsdevices/state.py#L700 but then I noticed that this file defines a `TWINCAT_MAX_STATES` and I shook my head

Reply:
That's deserving of a tsk-tsk and a frown alongside that head shake. Nothing is easy!